docs/markdown/concepts-and-architecture/components.md

```diff
@@ -0,0 +1,125 @@
+
+*These parachain commitments are effectively the sending part of a channel for communication from Polkadot to Ethereum. We may rename/refactor this functionality for consistency in future*
+
+For more details, see [Parachain Commitments](./polkadot-light-client-verifier/parachain-commitments)
+
+### <a name="polkadot-light-client-verifier"></a> Polkadot and Parachain Light Client Verifier
+For trustless verification on Ethereum we need a Polkadot light client and a parachain light client for our parachain. We have a set of smart contracts that run on Ethereum to provide this.
+
+For more details, see [Polkadot Light Client Verifier](./polkadot-light-client-verifier/)
+
+## Basic Bootstrap Bridge
+The basic bootstrap bridge is a lower level bridge intended to act as a bootstrap and simpler alternative to the main bridge. It is not incentivized and so does not guarantee delivery. It does however guarantee deliverability and replay protection.
+
+This bridge consists of 2 basic channels.
+
+### Basic Ethereum to Polkadot Message Channel
+The Basic Ethereum to Polkadot Message Channel is a channel for sending Polkadot RPCs out from Ethereum to Polkadot via events. It consists of a smart contract on the Ethereum side and a corresponding pallet on the parachain side. 
+
+The smart contract is responsible for accepting requests from other smart contracts for Ethereum RPCs to be sent over to Polkadot. It puts those requests into a channel that a corresponding pallet will receive on the Polkadot side.
+
+For more details, see [Basic Ethereum to Polkadot Message Channel](./basic-ethereum-to-polkadot-message-channel)
+
+### <a name="basic-polkadot-to-ethereum-message-channel"></a> Basic Polkadot to Ethereum Message Channel
+The Basic Polkadot to Ethereum Message Channel is a channel for sending Ethereum RPCs out from the parachain to Ethereum. It is responsible for accepting requests from other pallets and parachains for messages to be sent over to Ethereum. It puts those requests into [Parachain Message Commitments](#parachain-message-commitment) that will be included in the parachain header. The channel then processes those commitments and verifies them via the [Polkadot and Parachain Light Client Verifier](#polkadot-light-client-verifier) to extract Ethereum RPCs. Those Ethereum RPCs are then routed to their target contract by calling that contract.
+
+For more details, see [Basic Polkadot to Ethereum Message Channel](./basic-polkadot-to-ethereum-message-channel)
+
+## Incentivized Bridge
+The incentivized bridge extends the basic bootstrap bridge by adding a strict message ordering channels in both directions that enforce delivery of messages in the order that they are sent. It also adds incentives which provide guaranteed delivery with strong properties based on simple economic assumptions without the need for any kind of Oracle-based system.
```

Comment:
Can leave out the word "bootstrap" for less wordiness.

Also "Oracle-based solution" instead of "system"